apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -169,7 +169,7 @@ type MariadbSection struct {
 // GaleraSection defines the desired state of Galera services
 type GaleraSection struct {
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=true
+	// +kubebuilder:default=false
```

Comment:
according to https://github.com/openstack-k8s-operators/openstack-operator/pull/223, omitempty should be removed as well